andela/mrm_api
# #166189438 Add auto_cancelled and app_booking columns to events table

### What does this PR do?
 Modifies events model to have new columns `app_booking` and `auto_cancelled`.

### Description of the task to be completed?
- Modified events model to have new columns `app_booking` and `auto_cancelled`.

### How should this be manually tested?
- Pull this branch ch-add-app-bookings-and-auto-cancellation-columns-166189438
- Run `make migrate` to run migrations.
- Check the `events` table with a Postgres client like Postico, the new fields `app_booking` and `auto_cancelled` should exist.
#### Screenshots (if appropriate)

### What are the relevant pivotal tracker stories?
[#166189438](https://www.pivotaltracker.com/story/show/166189438)

#### Any background context that you would like to provide:
N/A
